### PR TITLE
Includes the maps page basics

### PR DESCRIPTION
**Pull request by AnnaKawakami**
Basics of map page includes the fundamental layout of the Google maps on the maps.html page, with the map defaulted to a marker pointing to Google headquarters.

### CODE REVIEW

**Comment by osjuga** [comment]
bash-profile(.save) and factorypath shouldn't be committed.

**Comment by osjuga** [comment]
Should probably git pull then create new branches as to not have old commits from master show up as new changes here and prevent small inconsistencies like the same code on different lines

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -43,15 +43,16 @@ public void storeMessage(Message message) {
...
     datastore.put(messageEntity);
   }
 
   /**
-   * Iterates through a result and return a list of messages
-   *
-   * @return list of messages
-   */
+  * Iterates through a result and return a list of messages
+  *
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/data/Datastore.java:53]: please correct indent here

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -73,67 +75,74 @@ public void storeMessage(Message message) {
...
  }
-
- /** 
-  * Gets the average length of all users' messages. Note -- still in progress.
-
-  @return a decimal number representing the average length of all
-  users' messages.
-
-
-public int getAvgMessageLength() {
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/data/Datastore.java:125]: please don't erase earlier methods

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -93,6 +114,7 @@ function buildMessageDiv(message) {
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
   setPageTitle();
-  showMessageFormIfViewingSelf();
+  showMessageFormIfLoggedIn();
```
Comment (gauravmishra) [inline src/main/webapp/js/user-page-loader.js:117]: i don't think this should be part of this pull

**Review by gauravmishra — CHANGES_REQUESTED** [review]
Great work! I think your pull request contains part of previous code added by other people because your local code repo wasn't synced before you created a new branch. Please use `git pull origin master` to sync your code repo before submitting a pull request.

**Pull request closed without merging**